packages/wonder-blocks-core/src/util/enumerate-scroll-ancestors.js

```diff
@@ -1,98 +0,0 @@
-// @flow
-/**
- * Utility for finding the scroll ancestors of a child
- *
- * Inspired by https://github.com/olahol/scrollparent.js
- * with modifications to make an iterator that returns a sequence of all scroll
- * ancestors.
- *
- * Also modified for our standards (and commented a bit).
- */
```

Comment:
Are we not using this any more?

Reply:
That's right. We are not importing this module anywhere in the codebase. This was probably some leftovers that I forgot to remove when we migrated from Popper v1 to v2.